src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,13 @@
 import React from 'react';
+import { Users } from '../../types/Users';
+import './UserInfo.scss';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
+type Props = {
+  user: Users;
```

Comment:
Why one user have type `Users`?
```suggestion
  user: User;
```